src/OSPSuite.Presentation/Presenters/SimulationOutputMappingPresenter.cs

```diff
@@ -23,7 +26,7 @@ public interface ISimulationOutputMappingPresenter : IPresenter<ISimulationOutpu
    {
       void EditSimulation(ISimulation simulation);
       IReadOnlyList<SimulationQuantitySelectionDTO> AllAvailableOutputs { get; }
-      void RemoveOutputMapping(SimulationOutputMappingDTO outputMappingDTO);
+      void RemoveObservedData(SimulationOutputMappingDTO outputMappingDTO);
```

Comment:
Hum...Aren't we remoeving an OutputMapping? Why did we rename this?

Reply:
when we click the "x" button we remove the observed data. to remove the output mapping, the <None> entry should be selected from the output dropdown.

Reply:
I see. Discutable logic but this is what the issue is about so I understand the change. Thanks